tests/acceptance/TestHelpers/CollaborationHelper.php

```diff
@@ -0,0 +1,50 @@
+<?php declare(strict_types=1);
+/**
+ * ownCloud
+ *
+ * @author Amrita Shrestha <amrita@jankaritech.com>
+ * @copyright Copyright (c) 2024 Amrita Shrestha amrita@jankaritech.com
+ */
+
+namespace TestHelpers;
+
+use GuzzleHttp\Exception\GuzzleException;
+
+/**
+ * A helper class for managing wopi requests
+ */
+class CollaborationHelper {
+
+	/**
+	 * @param string $fileId
+	 * @param string $app
+	 * @param string $username
+	 * @param string $password
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string|null $viewMode
+	 *
+	 * @return string
+	 * @throws GuzzleException
+	 */
+	public static function sendPOSTRequestOnOpenApp(
+		string $fileId,
+		string $app,
+		string $username,
+		string $password,
+		string $baseUrl,
+		string $xRequestId,
+		?string $viewMode,
+	): string {
+		$url = $baseUrl . "/app/open?app_name=$app&file_id=$fileId"
+			. ($viewMode !== null ? "&view_mode=$viewMode" : "");
+
+		return HttpRequestHelper::post(
+			$url,
+			$xRequestId,
+			$username,
+			$password,
+			['Content-Type' => 'application/json']
+		)->getBody()->getContents();
```

Comment:
just return the response
```suggestion
		);
```